src/App.tsx

```diff
@@ -1,109 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
Ensure that the `getNumbers` function is correctly implemented in `utils` and returns an array of numbers as expected. This function is crucial for generating the list of items.